webapp/channels/src/components/admin_console/server_logs/log_list.tsx

```diff
@@ -270,7 +270,7 @@ export default class LogList extends React.PureComponent<Props, State> {
         const placeholderEmpty: JSX.Element = (
             <FormattedMessage
                 id='admin.channel_settings.channel_list.no_channels_found'
```

Comment:
Unfortunately, this is not enough to accomplish the expected behaviour. 
This changes only the default message in case of translation is not found. You need to add a new value like `"admin.channel_settings.channel_list.no_logs_found": "No logs found",` in https://github.com/mattermost/mattermost/blob/bc11b298073033bcf9563ab1461d4000b9873e9e/webapp/channels/src/i18n/en.json. After that you can replace the `id` and `defaultMessage` like 

```suggestion
                id='admin.channel_settings.channel_list.no_logs_found'
```

After your changes dont forget to run `npm run i18n-extract` in `mattermost/webapp/channels` to properly order the new added translation.